client/my-sites/themes/style.scss

```diff
@@ -253,3 +273,30 @@
 		background: linear-gradient( to bottom, transparentize( white, 0.5 ), transparent );
 	}
 }
+
+@include breakpoint( ">1040px" ) {
```

Comment:
The theme screenshot column takes some space on the right, causing the left column to be a bit narrow.

I used `>1040px` to avoid this behavior:

![screen shot 2016-04-24 at 11 51 23 am](https://cloud.githubusercontent.com/assets/5881557/14768722/a73d0c7e-0a17-11e6-81cf-2ad09c858034.png)

I'm not sure if it's OK to have this though? Thoughts?